docs/explanations/pro_token_and_machine_usage.rst

```diff
@@ -0,0 +1,65 @@
+.. _pro_token_and_machine_usage:
+
+Machine counts and Token usage for Ubuntu Pro
+*********************************************
+
+Pro Tokens:
+-----------
+
+- Obtaining Tokens:
+    After subscribing to Ubuntu Pro, you can access your tokens through the Ubuntu Pro dashboard. Each token can be used to attach multiple machines depending on the subscription.
+
+- Attaching Tokens:
+    Command: Use ``sudo pro attach <token>`` to connect a machine to your Ubuntu Pro subscription.
+
+- Detaching Tokens:
+    Command: Use ``sudo pro detach`` to remove a machine from the subscription.
```

Comment:
Probably also worth mentioning here that users don't need to specifically detach a VM that they plan to destroy anyway, and that once they detach they can reuse the token